src/internal/operators/throttle.ts

```diff
@@ -61,9 +61,10 @@ export const defaultThrottleConfig: ThrottleConfig = {
  */
 export function throttle<T>(
   durationSelector: (value: T) => ObservableInput<any>,
-  { leading, trailing }: ThrottleConfig = defaultThrottleConfig
+  config: ThrottleConfig = defaultThrottleConfig
```

Comment:
This change is added just because we could have a better documentation. Instead of [this](https://rxjs.dev/api/operators/throttle):

![image](https://user-images.githubusercontent.com/28087049/117639371-21744700-b184-11eb-943c-4878d6c5c3cd.png)


we can have this:

![image](https://user-images.githubusercontent.com/28087049/117639154-e70aaa00-b183-11eb-8bcb-d2a8f9a8163d.png)